tests/highchart-wordcloud/spec.svg

```diff
@@ -0,0 +1,22 @@
+<svg version="1.1" class="highcharts-root" style="font-family:&quot;Lucida Grande&quot;, &quot;Lucida Sans Unicode&quot;, Arial, Helvetica, sans-serif;font-size:12px;" xmlns="http://www.w3.org/2000/svg" width="860.520720124746" height="400" viewBox="0 0 860.520720124746 400"><desc>Created with Highcharts 7.2.1</desc><defs><clipPath id="highcharts-8hytlhj-208-"><rect x="0" y="0" width="841" height="311" fill="none"></rect></clipPath></defs><rect fill="rgba(255,255,255,0.6)" class="highcharts-background" x="0" y="0" width="860.520720124746" height="400" rx="0" ry="0"></rect><rect fill="none" class="highcharts-plot-background" x="10" y="74" width="841" height="311"></rect><g class="highcharts-grid highcharts-xaxis-grid" data-z-index="1"></g><g class="highcharts-grid highcharts-yaxis-grid" data-z-index="1"></g><rect fill="none" class="highcharts-plot-border" data-z-index="1" x="10" y="74" width="841" height="311"></rect><g class="highcharts-axis highcharts-xaxis" data-z-index="2"><path fill="none" class="highcharts-axis-line" data-z-index="7" d="M 10 385 L 851 385"></path></g><g class="highcharts-axis highcharts-yaxis" data-z-index="2"><path fill="none" class="highcharts-axis-line" data-z-index="7" d="M 10 74 L 10 385"></path></g><g class="highcharts-series-group" data-z-index="3"><g data-z-index="0.1" class="highcharts-series highcharts-series-0 highcharts-wordcloud-series highcharts-tracker" transform="translate(430.5,229.5) scale(3.9196534068530915 3.9196534068530915)">
```

Comment:
Please reduce the test case to 4 cases
* `0,0`
* `x,0`
* `0,y`
* `x.y`
...and get rid of all other properties (style, opacity, ...)